Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /tikv
 
 # Install Rust
 COPY rust-toolchain ./
-RUN rustup default nightly-2019-07-19
+RUN rustup default nightly-2019-08-26
```

Comment:
do we need to upgrade the latest version now? @brson

Reply:
I can upgrade once more, but then this has got to land.

Reply:
This is done.